packages/ajax/stories/index.stories.mdx

```diff
@@ -48,15 +48,138 @@ ajax.get('./packages/ajax/stories/data.json').then(response => console.log(respo
 
 ### Installation
 
+Using npm:
+
 ```sh
 npm i --save @lion/ajax
 ```
 
+Using yarn:
```

Comment:
we only do npm... yarn users usually know how to map it 👍

Reply:
Removed 👍